src/sentry/static/sentry/app/components/timeSince.tsx

```diff
@@ -149,7 +166,7 @@ export function getRelativeDate(
       suffix,
     });
   } else if (shorten && !suffix) {
-    return getDuration(moment().diff(moment(date), 'seconds'), 0, true);
+    return getDuration(moment().diff(moment(date), 'seconds'), 0, true, extraShort);
```

Comment:
Doesn't this mean `extraShort` will always also need `shorten` to be `true` as well? I think this will easily cause confusion.

Reply:
@taylangocmen changed it to allow either, but if both are true you get extraShort.